languages/swift/exercises/concept/structs-and-classes/Tests/StructsAndClassesTests/StructsAndClassesTests.swift

```diff
@@ -0,0 +1,274 @@
+import XCTest
```

Comment:
I guess this test doesn't compile at the beginning because of missing structs and classes, right?
I was just asking this to myself with another exercise, if this is ok. The downside is, students can not solve tasks independently.

An alternative would be to comment all tests out except the one(s) for first task. This again has the downside, that students could forget to uncomment further tests later on.

What are your thoughts to this?

Reply:
Yeah, I'm not crazy about that setup, but unlike exercises which had just relied on functions, where I could throw a fatal error as the function body and everything would still compile, I don't really have that option with types, property access, and methods. It's either this, write two-thirds of the exercise for them, or, like you say, comment out the tests, which also has big downsides (although they already need to modify the test suite if they want to run more than the first test as it is).

I see this being an issue for most of the exercises from this point on that will be built around structs and enums, and I'm not sure there's a good solution.

Reply:
Thanks for your thoughts and would go with your solution in this case. In a current branch I went the way giving already the method definition, which is 1/2 - 2/3 of the solution :) Think I will change it as well and accept non-compiling tests.
(In case you want to have a look or review it, but no obligation ;) https://github.com/exercism/v3/pull/2205 )